### DISCUSSION TITLE
We have ingested all of our data, what next?

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/registry#88] [Q&A] [answered]
Now that we have ingested all of our data into the Registry, we are interested to know whether we have any further action items.  We will assume that we do not, unless and until we hear otherwise from you.

**Comment by pds-ops** [comment, marked as answer]
Congrats on ingesting all your data into the Registry! All of your node's data should now be accessible to the world.

### What next?
* You can try searching/analyzing your data using the [PDS Search API](https://nasa-pds.github.io/pds-api/)
* You can start to design new/adapt old apps to start to use the PDS Search API as their backend search service.
* You can submit new features requests or improvements you would like to the tools on [Github](https://github.com/NASA-PDS/registry/issues/new/choose).

### Further actions?
Not really actions as much as semi-regular checks to ensure the Registry Loader Tools (Harvest, Registry Manager, etc.):

* **Are included in your archive pipelines** - be sure to run the tools against any data updated/added to the archive.
* **Remain up-to-date** - Keep an eye out for release announcements sent to the pds-registry-users@groups.io email list, or you can follow Registry Releases on Github:
   * Go to the Github repo: https://github.com/NASA-PDS/registry/ 
   * Select Watch (in the top right corner) -> Custom -> Releases
   * <img width="344" alt="Screen Shot 2022-08-19 at 10 44 28 AM" src="https://user-images.githubusercontent.com/93455202/185656825-201712a1-5ce6-4930-acc5-e1404b328558.png">
